### PR TITLE
ROUTING-1182: adding cross-compilation instructions where missing for…

### PR DESCRIPTION
**Pull request by juanlu-rti**
… Infrastructure Services

### Summary

Several examples for Infrastructure Services were missing instructions for cross-compilation. I have added them where necessary.

### Checks

-   [ ] I have updated the documentation accordingly.
-   [ ] I have read the [CONTRIBUTING](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/CONTRIBUTING.md) document.

### CODE REVIEW

**Review by fgaranda — DISMISSED** [review]

File: examples/routing_service/file_adapter/c++11/RsFileAdapter.xml [hunk 1]
```diff
@@ -10,8 +10,8 @@
         <adapter_plugin name="FileAdapter">
             <!--
             By specifing this value, RTI Router will search for
-            libfileadapter.so in Unix systems, fileadapter.dll on
-            Windows systems and libfileadapter.dylib on Mac OS. 
+            libFileAdapterC++11.so in Unix systems, FileAdapterC++11.dll on
```
Comment (juanlu-rti) [inline examples/routing_service/file_adapter/c++11/RsFileAdapter.xml:13]: @fgallegosalido this changes are from one of your latest PRs right?

File: examples/routing_service/file_adapter/c/file_bridge.xml [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <!-- Adapter entry points are defined here -->
     <adapter_library name="adapters">
         <adapter_plugin name="file">
-            <dll>fileadapter</dll>
+            <dll>FileAdapterC</dll>
```
Comment (juanlu-rti) [inline examples/routing_service/file_adapter/c/file_bridge.xml:20]: @fgallegosalido same as above, these changes come from your PR, right?

**Review by Antoniolm — APPROVED** [review]

File: examples/routing_service/file_adapter/c++11/RsFileAdapter.xml (hunk 1 above)
Comment (juanlu-rti) [reply examples/routing_service/file_adapter/c++11/RsFileAdapter.xml:13]: Resolved this with Antonio L

File: examples/routing_service/file_adapter/c/file_bridge.xml (hunk 2 above)
Comment (juanlu-rti) [reply examples/routing_service/file_adapter/c/file_bridge.xml:20]: Resolved this with Antonio L

**Pull request merged**